### PR TITLE
[docs] Mention linters in Theia Tips & Tricks

### PR DESCRIPTION
**Pull request by jankeromnes**
As suggested by https://spectrum.chat/gitpod/general/how-can-i-configure-pylint-within-theia~92d41343-85c4-4bd4-81aa-0e9f441012d1?m=MTU2MjkyMjA4MTI3MA==

### CODE REVIEW

File: src/docs/52_Tips_and_Tricks.md [hunk 1]
```diff
@@ -65,3 +66,16 @@ you navigate between them in both directions:
 To tidy up your code, many languages support `Format Document` (<kbd>Alt+Shift+F</kbd>).
 If you want to apply this to only a subsection of the current document, select the desired region
 and pick `Format Selection` from the editor's context menu.
+
+## Linters
+
+You can [create a `setup.cfg` or a `pycodestyle.cfg` in the project root](https://github.com/palantir/python-language-server#configuration) and [adjust pycodestyle](http://pycodestyle.pycqa.org/en/latest/intro.html#configuration) rules there like this:
```
Comment (jankeromnes) [inline src/docs/52_Tips_and_Tricks.md:72]: Please change this

```suggestion
You can [write a `setup.cfg` or a `pycodestyle.cfg` in the project root](https://github.com/palantir/python-language-server#configuration) and [adjust pycodestyle](http://pycodestyle.pycqa.org/en/latest/intro.html#configuration) rules there like this:
```

**Comment by jankeromnes** [comment]
Hey @JesterOrNot, would this be interesting for the Python guide?

**Comment by JesterOrNot** [comment]
Yea I think so.

**Comment by jankeromnes** [comment]
@JesterOrNot Please include this in the Python guide when you have time. 🙏

**Comment by JesterOrNot** [comment]
It would be much easier to just create a new PR

**Comment by JesterOrNot** [comment]
closing in favor of #661

**Pull request closed without merging**